src/test/kotlin/no/nav/familie/ef/sak/cucumber/domeneparser/BasisDomeneParser.kt

```diff
@@ -233,11 +235,20 @@ fun parseBehandlingstype(rad: Map<String, String>): BehandlingType? {
     return parseValgfriEnum<BehandlingType>(SaksbehandlingDomeneBegrep.BEHANDLINGSTYPE, rad)
 }
 
+fun parseOppfølgingsoppgave(rad: Map<String, String>): VurderHenvendelseOppgavetype {
```

Comment:
Trengs denne? parseEnum burde kunne brukes direkte

Reply:
Og hvis den trengs kan den nok flyttes til en egen fil for SettPåVent-parsing :)

Reply:
Fjernet: [ce6fff7](https://github.com/navikt/familie-ef-sak/pull/2184/commits/ce6fff725e7da715baf68b034a915cca65f21ba6)